src/EditorFeatures/Core/ExternalAccess/VSTypeScript/VSTypeScriptEditorInlineRenameService.cs

```diff
@@ -24,6 +24,19 @@ internal sealed class VSTypeScriptEditorInlineRenameService(
 {
     private readonly Lazy<VSTypeScriptEditorInlineRenameServiceImplementation>? _service = service;
 
+     public bool IsEnabled
+     {
+        get
+        {
+            if (_service != null)
+            {
+                return _service.Value.IsEnabled();
+            }
+
+            return AbstractEditorInlineRenameService.DefaultEnablement;
```

Comment:
`DefaultEnablement` is only used here right?  If so I would just inline it.

Reply:
Will do